cluster-scope/overlays/nerc-ocp-prod/feature/external-console-access/api-proxy/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: api-proxy
+spec:
+  replicas: 3
+  template:
+    spec:
+      serviceAccountName: haproxy
+      containers:
+      - name: haproxy
+        image: docker.io/haproxy:lts
+        ports:
+        - containerPort: 8080
+          name: http
+        volumeMounts:
+          - name: haproxy-config
+            mountPath: /usr/local/etc/haproxy
+          - name: haproxy-state
+            mountPath: /var/lib/haproxy
+        securityContext:
+          allowPrivilegeEscalation: false
+          runAsUser: 0
```

Comment:
Why does this need to run as root?

Reply:
When I wrote this, I assumed that haproxy's default behavior of start as root and then suid-ing to an unprivileged user was intrinsic to haproxy, but it turns out that removing the `user` and `group` directories is sufficient.